docs/e2e.md

```diff
@@ -10,8 +10,13 @@ Pre-requisite for KinD clusters: please add in your local `/etc/hosts` file `127
   ```
   make kind-e2e
   export CLUSTER_HOSTNAME=kind
-  make deploy -e IMG=quay.io/project-codeflare/codeflare-operator:v1.1.0
   make setup-e2e
+
+  # For running tests locally on Kind cluster, we need to disable oauth (which is enabled by default) by setting the boolean value as false
+  [Disable oauth](https://github.com/project-codeflare/codeflare-operator/blob/main/main.go#L117)
```

Comment:
This can be done by modifying `codeflare-operator-config` ConfigMap, disabling `rayDashboardOAuthEnabled` and restarting CFO.